workflows/create_test_data.py

```diff
@@ -0,0 +1,69 @@
+"""
+This script handles the following:
+- Extract a subset of data from the `species.json` and `stations.json` files.
+- Load this data into separate `.json` files so that it can be used to create POSTGRES_TEST_DB
+"""
+
+import json
+import os
+import sys
+from typing import Union
+
+# Type Definition
+Data = Union[dict, list[dict]]
+
+# Define system specific paths
+species_json = os.path.join("data", "Oceans1876", "species.json")
+stations_json = os.path.join("data", "Oceans1876", "stations.json")
+ouput_species_json = os.path.join("data", "Oceans1876_subset", "test_species.json")
+ouput_stations_json = os.path.join("data", "Oceans1876_subset", "test_stations.json")
+
+# Define Cardinality of the Stations subset.
+N_stations = 15
+
+# Import JSON data
+def import_json(filename: str) -> Data:
+    try:
+        with open(filename) as jf:
+            data = json.load(jf)
+        return data
+    except FileNotFoundError:
+        sys.exit(f"{filename} is missing")
+
+
+# Import JSON data
+def export_json(filename: str, output: Data) -> None:
+    try:
+        os.makedirs(os.path.join("data", "Oceans1876_subset"), exist_ok=True)
+        with open(filename, "w") as ojf:
+            json.dump(output, ojf, indent=4)
+    except Exception as e:
+        print(e.with_traceback())
+
+
+# Species only
+species_data = import_json(species_json)[
```

Comment:
Let's move the following lines and what's after line 57 in a function and call the function after the `if __name__...` statement.

This works when we run the script directly (`python workflows/create_test_data.py`), but if we import it into another module, we won't be able to run the code after line 57.